assets/postit-js.css

```diff
@@ -90,4 +167,306 @@
   background-color: black !important;
 }
 
+/* screen interaction helpers & modal */
+
+.io-editing-tools {
```

Comment:
Can we prefix all css classes we use in the core with `pjs-`? Just to make it consistent. Also with the buttons and so on above